src/reactions/reactions.controller.ts

```diff
@@ -13,4 +16,51 @@ export class ReactionsController {
 
         res.json(await ReactionsService.create(req.body, req.headers.authorization!));
     }
+
+    static async getReactionAmountByTypeAndResourceType(req: Request, res: Response) {
+        const resourceReactions: any[] = await ReactionsService.getReactionAmountByTypeAndResourceType(req.query, req.headers.authorization!);
+
+        res.json(await ReactionsController.popolateVideos(resourceReactions, req.user));
+    }
+
+    static async getUserLikedVideos(req: Request, res: Response) {
+        const reactions: any[] = await ReactionsService.getMany(req.query, req.headers.authorization!);
+
+        res.json(await ReactionsController.popolateVideos(reactions, req.user));
+    }
+
+    private static async popolateVideos(reactions: any[], user: any) {
```

Comment:
spelling mistake: change popolate to populate.

Reply:
Fixed